### PR TITLE
Feat/ab#104252 eis map widget layers should always appear based on the order admins configured them

### PR DESCRIPTION
**Pull request by unai-reliefapp**
# Description

The idea is to use the [pane property from leaflet](https://leafletjs.com/examples/map-panes/) to easily order current layer stack no matter the layer types.

We set the zIndex of each layer pane from 500 to above, this way we make sure is keeps the correct layer stack order and that there is no conflict with other map features, like popups

## Useful links

- [Link to ticket](https://dev.azure.com/WHOHQ/EMSSAFE/_workitems/edit/104252)
- Please insert link to back-end branch if any
- Please insert any useful link ( documentation you used for example )

## Type of change

Please delete options that are not relevant.

- [X] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

- [ ] Test A
- [ ] Test B

## Screenshots

Stack order set for layers in a map
https://github.com/user-attachments/assets/06128103-c4da-42d8-9892-eeca55f425b5

Stack order set for layers between points and polygons too
https://github.com/user-attachments/assets/c1ad3427-f1fc-4055-837e-f934893d5016

Stack order set for sublayers in a group layer
https://github.com/user-attachments/assets/c3372f49-12c2-41d6-b60e-033d6fd2aac0

Stack order set for between layers and group layers
https://github.com/user-attachments/assets/dbc4663f-b293-4e51-88a8-b19e83d82fee

Info pills
image
image

# Checklist:

( * == Mandatory ) 

- [X] * I have set myself as assignee of the pull request
- [X] * My code follows the style guidelines of this project
- [X] * Linting does not generate new warnings
- [X] * I have performed a self-review of my own code
- [X] * I have put the ticket for review, adding the oort-frontend team to the list of reviewers
- [X] * I have commented my code, particularly in hard-to-understand areas
- [X] * I have put JSDoc comment in all required places
- [X] * My changes generate no new warnings
- [X] * I have included screenshots describing my changes if relevant
- [X] * I have selected labels in the Pull Request, according to the changes with code brings
- [ ] I have made corresponding changes to the documentation ( if required )
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [X] Any dependent changes have been merged and published in downstream modules

# More explanation
https://www.loom.com/share/05a716d61b9744faaf51fb304c21d1e5?sid=f87cf896-582a-4f76-93ae-8ceed801b145

### CODE REVIEW

**Comment by AntoineRelief** [comment]
@unai-reliefapp 

( Please don't update the code, I'll do the according changes to the help text, but I only need to clarify with you )

> Layer stack order on the map would be applied based on the layer list

For me, this does not correctly indicate what is the order ( ascending or descending ) 

Do the layers on top have the higher index, or the opposite?

**Comment by unai-reliefapp** [comment]
> @unai-reliefapp
> 
> ( Please don't update the code, I'll do the according changes to the help text, but I only need to clarify with you )
> 
> > Layer stack order on the map would be applied based on the layer list
> 
> For me, this does not correctly indicate what is the order ( ascending or descending )
> 
> Do the layers on top have the higher index, or the opposite?

@AntoineRelief You are right, the order is the one in the top of the list would be in the top of the stack(higher index) 👍 
I added some videos in the PR description to check how it behaves

**Comment by AntoineRelief** [comment]
@unai-reliefapp 

technically speaking, it's all good!

I just have a question about the git history
the requested changes are fine, and only the correct ones are in the PR
but for some reasons, I have about 18 commits in the PR
did you create the branch from incorrect commit then reverted them?

**Comment by unai-reliefapp** [comment]
> @unai-reliefapp
> 
> technically speaking, it's all good!
> 
> I just have a question about the git history the requested changes are fine, and only the correct ones are in the PR but for some reasons, I have about 18 commits in the PR did you create the branch from incorrect commit then reverted them?

As far as I know i used beta as base branch, let me rebase it and check it again
I know what happened @AntoineRelief , my local beta branch was set months ago, and it's just take it from local and update with remote, let me fix that

File: libs/shared/src/i18n/fr.json [hunk 1]
```diff
@@ -1881,7 +1881,8 @@
 						}
 					},
 					"help": {
-						"geographicExtent": "Utilisez le filtre et/ou le contexte de la page pour zoomer sur la carte"
+						"geographicExtent": "Utilisez le filtre et/ou le contexte de la page pour zoomer sur la carte",
+						"layers": "*******************"
```
Comment (AntoineRelief) [inline libs/shared/src/i18n/fr.json]: @unai-reliefapp 
perfect 👍 
like that it's super clear

**Comment by AntoineRelief** [comment]
> > @unai-reliefapp
> > technically speaking, it's all good!
> > I just have a question about the git history the requested changes are fine, and only the correct ones are in the PR but for some reasons, I have about 18 commits in the PR did you create the branch from incorrect commit then reverted them?
> 
> As far as I know i used beta as base branch, let me rebase it and check it again I know what happened @AntoineRelief , my local beta branch was set months ago, and it's just take it from local and update with remote, let me fix that

@unai-reliefapp 
it's fine, don't spend more time on that
In any case, I'm reducing the PR to a single commit, so I was just curious about why history would display like that

**Comment by unai-reliefapp** [comment]
> > > @unai-reliefapp
> > > technically speaking, it's all good!
> > > I just have a question about the git history the requested changes are fine, and only the correct ones are in the PR but for some reasons, I have about 18 commits in the PR did you create the branch from incorrect commit then reverted them?
> > 
> > 
> > As far as I know i used beta as base branch, let me rebase it and check it again I know what happened @AntoineRelief , my local beta branch was set months ago, and it's just take it from local and update with remote, let me fix that
> 
> @unai-reliefapp it's fine, don't spend more time on that In any case, I'm reducing the PR to a single commit, so I was just curious about why history would display like that

Noice, I already fix that, I open PR for ticket 104241 and it's fine 👍

**Pull request merged**